### PR TITLE
Experiment Scripts

### PR DESCRIPTION
**Pull request by shokiami**
Bash scripts for experiments 3.1, 3.2, and 3.3.

### CODE REVIEW

File: experiments/draft.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+python ../visualization_utils/analyze_results.py $session_name"_3" $model_save_folder $visualizations_path
+python ../visualization_utils/analyze_results.py $session_name"_4" $model_save_folder $visualizations_path
+# visualize mistakes
+python ../visualization_utils/visualize_mistakes.py $session_name"_1" $image_base_path $crop_size $visualizations_path $num_plots
+python ../visualization_utils/visualize_mistakes.py $session_name"_2" $image_base_path $crop_size $visualizations_path $num_plots
+python ../visualization_utils/visualize_mistakes.py $session_name"_3" $image_base_path $crop_size $visualizations_path $num_plots
+python ../visualization_utils/visualize_mistakes.py $session_name"_4" $image_base_path $crop_size $visualizations_path $num_plots
+
+# remove temporary binarized train sets
+rm $csv_base_path"train_set_1.csv"
```
Comment (ThatOneGoat) [inline experiments/draft.sh:58]: What do you think about keeping the train/test sets in some sort of organized folder by experiments

Comment (shokiami) [reply experiments/draft.sh]: Gotchu bro

**Review by LoganMilandin — APPROVED** [review]
Solid! Has this been tested?

**Comment by shokiami** [comment]
@LoganMilandin Yuh, with a mini train/test set. Although don't worry about merging this PR yet (hence the draft PR), experiment 3.2 is incoming!

**Pull request merged**